src/test/java/org/cryptomator/cryptolib/v1/FileContentCryptorImplTest.java

```diff
@@ -91,7 +88,7 @@ public class Encryption {
 
 		@DisplayName("encrypt chunk with invalid size")
 		@ParameterizedTest(name = "cleartext size: {0}")
-		@ValueSource(ints = {0, Constants.PAYLOAD_SIZE + 1})
+		@ValueSource(ints = {Constants.PAYLOAD_SIZE + 1})
```

Comment:
@overheadhunter Should `0` be removed for v2/v3 as well, why the asymmetry? The corresponding test in v2 currently throws `IllegalArgumentException` not with `Invalid cleartext chunk size` as in `v1` but from different root cause (see screenshot) - is this expected or does this happen accidentally?

![grafik](https://github.com/user-attachments/assets/f1e1506b-dc5a-4f5d-a657-2cce6a4ccbe7)